src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -590,9 +626,28 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode): void
                                                         ->ifArray()
                                                         ->then(static fn ($to) => array_map(static fn ($v) => $v instanceof \BackedEnum ? $v->value : $v, $to))
                                                     ->end()
+                                                    ->beforeNormalization()
+                                                        ->always()
```

Comment:
new best practice: be more specific, so that the normalization can fail on its own when an unexpected type is met (together with acceptAndWrap() to accept some types:
```suggestion
                                                        ->ifArray()
```